query-creator/create-query/steps/aws.js

```diff
@@ -156,13 +156,16 @@ const getColumnsForQuestion = (questionId, question, denormalizedResource, authD
         `'${process.env.PORTAL_REPORT_URL}`,
         `?auth-domain=${encodeURIComponent(authDomain)}`,
         `&firebase-app=${process.env.FIREBASE_APP}`,
+        `&sourceKey=${sourceKey}`,
         `&iframeQuestionId=${questionId}`,
         `&class=${encodeURIComponent(`${authDomain}/api/v1/classes/`)}',`,
         ` CAST(class_id AS VARCHAR), `,
         `'&offering=${encodeURIComponent(`${authDomain}/api/v1/offerings/`)}',`,
         ` CAST(offering_id AS VARCHAR), `,
         `'&studentId=',`,
-        ` CAST(user_id AS VARCHAR)`,
+        ` CAST(user_id AS VARCHAR), `,
+        `'&answersSourceKey=', `,
+        ` source_key['${questionId}']`,
         `)`
       ].join("")
```

Comment:
I tried before to clean this up the syntax here but couldn't find any better way to write this.  The confusing part is that we are essentially doing concatenation both in javascript and SQL. 

Perhaps the most clear approach is a multiline whitespace stripping function. 
Another option is a to use SQL concat for every parameter and value.  That would probably look the most clean, but it would also be the least efficient for Athena to process (unless we we made it complex enough to join strings sometimes with javascript and concat them other times.

One of my previous attempts is here: https://github.com/concord-consortium/report-service/pull/92#discussion_r677585189